docs/content/direct/pr-signing.md

```diff
@@ -0,0 +1,119 @@
+   - Make sure to list your email as the same one that's verified by Github
+5. Use the following command to list the long form of your generated GPG keys:
+   ```shell
+   gpg --list-secret-keys --keyid-format=long
+   ```
+   - Your GPG key ID should be the characters on the output line starting with ```sec```, beginning directly after the ```/``` and ending before the listed date.
+     - For example, in the output below (from the Github [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key) setup page), the GPG key ID would be ```3AA5C34371567BD2```
+     ```shell
+     $ gpg --list-secret-keys --keyid-format=long
+      /Users/hubot/.gnupg/secring.gpg
+      ------------------------------------
+      sec   4096R/3AA5C34371567BD2 2016-03-10 [expires: 2017-03-10]
+      uid                          Hubot <hubot@example.com>
+      ssb   4096R/4BB6D45482678BE3 2016-03-10
+
+     ```
+6. Copy your GPG key ID and run the command below, replacing ```[your_GPG_key_ID]``` with the key ID you just copied:
+   ```shell
+   gpg --armor --export [your_GPG_key_ID]
+   ```
+7. This should generate an output with your GPG key. Copy the characters starting from ```-----BEGIN PGP PUBLIC KEY BLOCK-----``` and ending at ```--END PGP PUBLIC KEY BLOCK-----``` (inclusive) to your clipboard.
+8. After copying or saving your GPG key, navigate to **Settings** in your Github
+9. Navigate to the **SSH and GPG keys** page under the Access section in the sidebar
+10. Under GPG keys, select **New GPG key**
+    - Enter a suitable name for your key under "Title" and paste your GPG key that you copied/saved in **Step 7** under "Key".
+    - Once done, click **Add GPG key**
+11. Your new GPG key should now be displayed under GPG keys.
+
+<br />
+
+## Setting up the SSH Key
+
+1. Open Git Bash and use the following command to generate your new SSH key (make sure to replace ```your_email``` with your Github-verified email address):
+   ```shell
+   ssh-keygen -t ed25519 -C "your_email"
+   ```
+   
+2. Press ```Enter``` to select the default option if prompted to set a save-file or passphrase for the key (you may choose to enter a passphrase if desired; this will prompt you to enter the passphrase everytime you perform a DCO sign-off).
+   - The following output should generate a randomart image 
+3. Use the following command to copy the new SSH key to your clipboard:
+   ```shell
+   clip < ~/.ssh/id_ed25519.pub
+   ```
+4. After copying or saving your SSH key, navigate to **Settings** in your Github.
+5. Navigate to the **SSH and GPG keys** page under the Access section in the sidebar.
+6. Under SSH keys, select **New SSH key**.
+   - Enter a suitable name for your key under "Title"
+   - Open the dropdown menu under "Key type" and select **Signing Key**
+   - Paste your SSH key that you copied/saved in **Step 3** under "Key"
+7. Your new SSH key should now be displayed under SSH keys.
+
```

Comment:
please include how to test if your ssh key is working using "ssh" - check with https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection